pages/contact-info/commonContactInfo.tsx

```diff
@@ -0,0 +1,574 @@
+      } finally {
+        setLoading(false)
+      }
+    }
+    return (
+      <Layout data={data} title="Contact Information">
+        {isDisabled ? (
+          <ContactInfo active={true} />
+        ) : (
+          <ContactInfo active={false} />
+        )}
+        <h3 className="h5 mb-8">Residential Address</h3>
+        <form onSubmit={handleSubmit} noValidate>
+          <div className="flex-auto sm:w-2/3 md:max-w-xl lg:w-1/2 ml-2">
+            <Input
+              type="text"
+              name="streetNumber"
+              label="Street Number"
+              error={steetNumberError}
+              value={streetNumber}
+              onChange={(e) => setStreetNumber(e.target.value)}
+              placeholder="Enter Street Number"
+              required
+              autoFocus
+              disabled={isDisabled}
+            />
+            <Input
+              type="text"
+              name="streetName"
+              label="Street Name"
+              error={steetNameError}
+              value={streetName}
+              onChange={(e) => setStreetName(e.target.value)}
+              placeholder="Enter Street Name"
+              required
+              disabled={isDisabled}
+            />
+            <Input
+              type="text"
+              name="aptNumber"
+              label="Apt Number"
+              error={aptNumberError}
+              value={aptNumber}
+              onChange={(e) => setAptNumber(e.target.value)}
+              placeholder="Enter Apartment Number"
+              disabled={isDisabled}
+            />
+            <Input
+              type="text"
+              name="postalCode"
+              label="Postal Code"
+              error={postalCodeError}
+              value={postalCode}
+              onChange={(e) => setPostalCode(e.target.value)}
+              placeholder="Enter Postal Code"
+              required
+              disabled={isDisabled}
+            />
+            <Input
+              type="text"
+              name="city"
+              label="City"
+              error={cityError}
+              value={city}
+              onChange={(e) => setCity(e.target.value)}
+              placeholder="Enter City"
+              required
+              disabled={isDisabled}
+            />
+            <section>
+              {countryError ? <ErrorLabel errorMessage={countryError} /> : ''}
+              <label>
+                <span className="text-danger">*</span> Country
+                <span className="text-danger"> (required)</span>
+              </label>
+              <Select
+                name="country"
+                options={options}
+                value={options.filter((option) => option.value === country)}
+                onChange={changeCountryHandler}
+                isClearable
+                isDisabled={isDisabled}
+                placeholder="Select your country of residence"
+              />
+            </section>
+            <br></br>
```

Comment:
You shouldn't use line break tags for spacing, use margins instead

Reply:
I will change this. I added them before I learned about margins.